Navaids/FIXES_UK.txt

```diff
@@ -206,6 +206,7 @@ DIPSO N053.03.13.000 W001.20.39.000
 DISIT N052.06.11.000 W001.42.41.000
 DISVO N050.39.31.000 W001.02.57.000
 DITOB N052.17.01.000 E001.49.27.000
+DIXIB N051.14.12.000 W000.50.54.000
```

Comment:
Same question as before, are these going to be Airway fixes and in ENR 4.4 or defined only for the STAR?

Reply:
They're all in ENR 4.4 but it seems to be only for the STAR so.. I'm not sure

Reply:
I think these fixes are in the right place, there are also other fixes in ENR 4.4 that are only for STAR procedures defined in Fixes_UK.txt